plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/BuildPath.java

```diff
@@ -80,9 +80,10 @@ public static void addMavenLibraries(IProject project, List<Library> libraries,
     
     SubMonitor subMonitor = SubMonitor.convert(monitor,
         Messages.getString("adding.app.engine.libraries"), //$NON-NLS-1$
-        2);
+        3);
     
     Library masterLibrary = collectLibraryFiles(javaProject, libraries);
```

Comment:
Would it make sense to pass a monitor into `collectLibraryFiles()` and `computeEntries()`?